line-colors.csv

```diff
@@ -1,185 +1,190 @@
-vrs-stadtbahn,15,,8-vrs001-15,#5aad31,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,16,,8-vrs001-16,#07ada5,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,17,,8-vrs001-17,#1e93d1,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,18,,8-vrs001-18,#85d1f5,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,61,stadtwerke-bonn,8-s3-61,#9ab831,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,62,stadtwerke-bonn,8-s3-62,#65a53b,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,63,stadtwerke-bonn,8-s3-63,#8bccf3,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,65,stadtwerke-bonn,8-s3-65,#c6cc2a,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,66,stadtwerke-bonn,8-s3-66,#d4417d,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,67,stadtwerke-bonn,8-s3-67,#e7a8c4,#ffffff,,rectangle-rounded-corner
-vrs-stadtbahn,68,stadtwerke-bonn,8-s3-68,#cbaece,#ffffff,,rectangle-rounded-corner
-vvo-db-sbd,S 1,db-regio-ag-sudost,4-800469-1,#4f9551,#ffffff,,pill
-vvo-db-sbd,S 2,db-regio-ag-sudost,4-800469-2,#f58220,#ffffff,,pill
-vvo-db-sbd,S 3,db-regio-ag-sudost,4-800469-3,#00b3ef,#ffffff,,pill
-vvs-db-sbs,S1,db-regio-ag-s-bahn-stuttgart,4-800643-1,#59af41,#ffffff,,rectangle
-vvs-db-sbs,S11,db-regio-ag-s-bahn-stuttgart,4-800643-11,#59af41,#ffffff,,rectangle
-vvs-db-sbs,S2,db-regio-ag-s-bahn-stuttgart,4-800643-2,#ee1c28,#ffffff,,rectangle
-vvs-db-sbs,S3,db-regio-ag-s-bahn-stuttgart,4-800643-3,#f36e31,#ffffff,,rectangle
-vvs-db-sbs,S4,db-regio-ag-s-bahn-stuttgart,4-800643-4,#0166b3,#ffffff,,rectangle
-vvs-db-sbs,S5,db-regio-ag-s-bahn-stuttgart,4-800643-5,#00acdd,#ffffff,,rectangle
-vvs-db-sbs,S6,db-regio-ag-s-bahn-stuttgart,4-800643-6,#844c00,#ffffff,,rectangle
-vvs-db-sbs,S60,db-regio-ag-s-bahn-stuttgart,4-800643-60,#8a8d06,#ffffff,,rectangle
-vvs-db-sbs,S62,db-regio-ag-s-bahn-stuttgart,4-800643-62,#c3792e,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U1,,8-vvs020-u1,#cd9c72,#000000,,rectangle
-vvs-ssb-stadtbahn,U11,,8-vvs020-u11,#939598,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U12,,8-vvs020-u12,#85bbe5,#000000,,rectangle
-vvs-ssb-stadtbahn,U13,,8-vvs020-u13,#f59eb2,#000000,,rectangle
-vvs-ssb-stadtbahn,U14,,8-vvs020-u14,#69bd51,#000000,,rectangle
-vvs-ssb-stadtbahn,U15,,8-vvs020-u15,#015aaa,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U2,,8-vvs020-u2,#f36e31,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U29,,8-vvs020-u29,#fed304,#000000,,rectangle
-vvs-ssb-stadtbahn,U3,,8-vvs020-u3,#946241,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U34,,8-vvs020-u34,#69bd51,#000000,,rectangle
-vvs-ssb-stadtbahn,U4,,8-vvs020-u4,#7a67ae,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U5,,8-vvs020-u5,#00baf1,#000000,,rectangle
-vvs-ssb-stadtbahn,U6,,8-vvs020-u6,#ed028c,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U7,,8-vvs020-u7,#0ab38d,#ffffff,,rectangle
-vvs-ssb-stadtbahn,U8,,8-vvs020-u8,#fed304,#000000,,rectangle
-vvs-ssb-stadtbahn,U9,,8-vvs020-u9,#bfb678,#000000,,rectangle
+shortOperatorName,lineName,hafasOperatorCode,hafasLineId,backgroundColor,textColor,borderColor,shape

```

Comment:
Kannst du deine Änderungen bitte in der richtigen Reihenfolge (alphabetisch sortiert) speichern?

`vvs-ssb-stadtbahn should be after vrr-rrx-regional`

Reply:
VVS Stadtbahn hatte ich nicht angefasst 🤔

Reply:
Eben. :D

`vvs-rrx-` würde alphabetisch vor `vvs-ssb-` kommen. Du hast es aber danach eingetragen.